tests/bsim/_generate_coverage_report.sh

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env bash
+# Copyright 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+function display_help(){
+  echo "\
+_generate_coverage_report.sh [-help]
+  Generate an html coverage report for BabbleSim tests
+
+  You can call it from the Zephyr top level as:
+    tests/bsim/_generate_coverage_report.sh
+
+  Coverage files will be searched for in the folder pointed by
+  the variable WORK_DIR which by default is
+  ${ZEPHYR_BASE}/bsim_out
```

Comment:
Will the script generate coverage only for mesh if my `bsim_out` has only mesh compilation result? Or I need specify mesh path in any case?

Reply:
>Will the script generate coverage only for mesh if my bsim_out has only mesh compilation result

Yes. It just collects the coverage data that is there. So if you only run mesh tests you only get mesh tests coverage data generated, and only that processed into the report.